_extras/additional_exercises.md

```diff
@@ -145,7 +145,7 @@ Learner profiles have three parts:
 > "Issues" tab. Read through some of the discussions and, if you have anything to add, please add it
 > to the conversation! If you wish to make a pull request, be sure to examine the contribution guidelines for
 > the repository you are working in. If you do make a significant contribution to the discussion, send a link to
-> the issue to [checkout@carpentries.org](mailto: checkout@carpentries.org). Congratulations! You have
+> the issue to [checkout@carpentries.org](mailto:checkout@carpentries.org). Congratulations! You have
```

Comment:
@karenword  is this the right address?

Reply:
```suggestion
> the issue to [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org). Congratulations! You have
```

Reply:
Spoke with @karenword async and she says this is the correct email.